oak-search-elastic/src/test/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticIndexAndTraversalQueriesSimilarResultsCommonTest.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.jackrabbit.oak.plugins.index.elastic;
+
+import com.google.common.collect.ImmutableList;
+import org.apache.jackrabbit.oak.InitialContentHelper;
+import org.apache.jackrabbit.oak.api.ContentRepository;
+import org.apache.jackrabbit.oak.plugins.index.IndexAndTraversalQueriesSimilarResultsCommonTest;
+import org.apache.jackrabbit.oak.plugins.memory.MemoryNodeStore;
+import org.junit.ClassRule;
+
+public class ElasticIndexAndTraversalQueriesSimilarResultsCommonTest extends IndexAndTraversalQueriesSimilarResultsCommonTest {
+
+    @ClassRule
+    public static final ElasticConnectionRule elasticRule =
+            new ElasticConnectionRule(ElasticTestUtils.ELASTIC_CONNECTION_STRING);
+
+    public ElasticIndexAndTraversalQueriesSimilarResultsCommonTest() {
+        indexOptions = new ElasticIndexOptions();
+        super.passingQueries = ImmutableList.of(
```

Comment:
This and LuceneIndexAndTraversalQueriesSimilarResultsCommonTest seem to have the same values for passingQueries and failingQueries... so could we move this to IndexAndTraversalQueriesSimilarResultsCommonTest ? Maybe I'm missing some aspect that wouldn't allow this... or do you think this will change in the future?

Reply:
I moved these lists to the base class to remove duplication. In the future, I expect that they may be different, but then we can move the lists to the derived classes. For the time being, I agree it's better to avoid duplication.